index.js

```diff
@@ -1,22 +1,10 @@
 'use strict'
 
-var extend = require('extend')
-var bail = require('bail')
-var vfile = require('vfile')
-var trough = require('trough')
-var plain = require('is-plain-obj')
-
-// Expose a frozen processor.
-module.exports = unified().freeze()
-
-var slice = [].slice
-var own = {}.hasOwnProperty
-
-// Process pipeline.
-var pipeline = trough()
-  .use(pipelineParse)
-  .use(pipelineRun)
-  .use(pipelineStringify)
+import extend from 'extend'
+import bail from 'bail'
+import vfile from 'vfile'
+import trough from 'trough'
+import plain from 'is-plain-obj'
 
 function pipelineParse(p, ctx) {
```

Comment:
For some reason coverage doesn’t see this function as covered?

Reply:
This is likely also related to bcoe/c8#66